spell-check-custom-words.txt

```diff
@@ -50,6 +51,8 @@ Sway
 TAI
 TODO
 TypeScript
+UpgradePurpose
+UpgradePurposeType
```

Comment:
nit: these should probably just be in backticks (`code`) everywhere, so we don't need to spellcheck-allow them